open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    def get_the_unique_values_of_Pclass(df):
</pre><p>Please add some documentation for this function. And don't have upper case for python method.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>ok</p>